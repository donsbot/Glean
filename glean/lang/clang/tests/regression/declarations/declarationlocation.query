query: |
    { N, L } where cxx1.DeclarationLocationName { _, N, L }
transform: [gensort, []]
