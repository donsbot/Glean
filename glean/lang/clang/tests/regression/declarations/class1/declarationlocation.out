[
  "@generated",
  {
    "key": {
      "tuplefield0": {
        "file": { "key": "test.cpp" },
        "lineBegin": 9,
        "columnBegin": 1,
        "lineEnd": 9,
        "columnEnd": 11
      },
      "tuplefield1": "T"
    }
  },
  {
    "key": {
      "tuplefield0": {
        "file": { "key": "test.cpp" },
        "lineBegin": 10,
        "columnBegin": 1,
        "lineEnd": 14,
        "columnEnd": 1
      },
      "tuplefield1": "U"
    }
  },
  {
    "key": {
      "tuplefield0": {
        "file": { "key": "test.cpp" },
        "lineBegin": 11,
        "columnBegin": 3,
        "lineEnd": 11,
        "columnEnd": 5
      },
      "tuplefield1": "constructor"
    }
  },
  {
    "key": {
      "tuplefield0": {
        "file": { "key": "test.cpp" },
        "lineBegin": 12,
        "columnBegin": 3,
        "lineEnd": 12,
        "columnEnd": 13
      },
      "tuplefield1": "constructor"
    }
  },
  {
    "key": {
      "tuplefield0": {
        "file": { "key": "test.cpp" },
        "lineBegin": 12,
        "columnBegin": 5,
        "lineEnd": 12,
        "columnEnd": 12
      },
      "tuplefield1": ""
    }
  },
  {
    "key": {
      "tuplefield0": {
        "file": { "key": "test.cpp" },
        "lineBegin": 13,
        "columnBegin": 3,
        "lineEnd": 13,
        "columnEnd": 24
      },
      "tuplefield1": "operator="
    }
  },
  {
    "key": {
      "tuplefield0": {
        "file": { "key": "test.cpp" },
        "lineBegin": 13,
        "columnBegin": 16,
        "lineEnd": 13,
        "columnEnd": 23
      },
      "tuplefield1": ""
    }
  },
  {
    "key": {
      "tuplefield0": {
        "file": { "key": "test.cpp" },
        "lineBegin": 16,
        "columnBegin": 1,
        "lineEnd": 18,
        "columnEnd": 1
      },
      "tuplefield1": "A"
    }
  },
  {
    "key": {
      "tuplefield0": {
        "file": { "key": "test.cpp" },
        "lineBegin": 17,
        "columnBegin": 3,
        "lineEnd": 17,
        "columnEnd": 18
      },
      "tuplefield1": "destructor"
    }
  },
  {
    "key": {
      "tuplefield0": {
        "file": { "key": "test.cpp" },
        "lineBegin": 20,
        "columnBegin": 1,
        "lineEnd": 22,
        "columnEnd": 1
      },
      "tuplefield1": "B"
    }
  },
  {
    "key": {
      "tuplefield0": {
        "file": { "key": "test.cpp" },
        "lineBegin": 21,
        "columnBegin": 3,
        "lineEnd": 21,
        "columnEnd": 5
      },
      "tuplefield1": "u"
    }
  }
]